wallet/db.c

```diff
@@ -909,7 +909,7 @@ static struct migration dbmigrations[] = {
 	 ", resolved_time BIGINT"
 	 ", failcode INTEGER"
 	 ", forward_style INTEGER"
-	 ", PRIMARY KEY(in_channel_scid, in_htlc_id))"), NULL},
```

Comment:
This will break anybody who already upgraded to a version that included this migration

Reply:
For a person that already upgraded, this code is run again?

Keep this as is do not solve the problem? or I'm missing something in the logic?

There is a better way to fix this problem?